### DISCUSSION TITLE
Support Alerts in Loki Logs Visualization Panels

### DISCUSSION THREAD

**Discussion by mattparkes** [grafana/grafana#59446] [Dashboards, Panels & Visualizations]
I'd love to be able to put alerts on Logs visualization dashboard panels. The most common use case would be a panel with a query like:

```
{cluster="foo"} |= "error"
```

And then I want to be alerted when some function (e.g `avg()`, `max()`, etc) of `count_over_time` exceeds some threshold.

The UI I'm thinking would be identical to the `Time Series` Alert tab and would say something like

```
WHEN      <function>      OF      count_over_time(query(a, 1m, now))      IS WHATEVER      [ input field ]
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
